src/Util.cpp

```diff
@@ -588,15 +592,41 @@ void load_plugin(const std::string &lib_name) {
                    << "LoadLibraryW failed with error " << error << ": "
                    << err_msg << "\n";
     }
+
+    FARPROC as_name = GetProcAddress(library, "AutoschedulerName");
```

Comment:
I don't know I should be surprised that the legacy of NEAR/FAR ptrs on x86 lingers on to this day, but somehow I am